packages/orbit-components/src/Radio/index.tsx

```diff
@@ -29,25 +30,36 @@ const Radio = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
       htmlFor={id}
       className={cx(
         "font-base text-form-element-label-foreground relative flex w-full [align-items:self-start]",
-        disabled ? "cursor-not-allowed" : "cursor-pointer",
-        !disabled && [
-          !checked &&
-            (hasError
-              ? "[&>.orbit-radio-icon-container]:border-form-element-error [&>.orbit-radio-icon-container]:hover:border-form-element-error-hover [&>.orbit-radio-icon-container]:active:border-form-element-error"
-              : "[&>.orbit-radio-icon-container]:border-form-element [&>.orbit-radio-icon-container]:hover:border-form-element-hover [&>.orbit-radio-icon-container]:active:border-form-element-active"),
-          checked &&
-            "[&>.orbit-radio-icon-container]:border-form-element-focus active:border-form-element-focus [&>.orbit-radio-icon-container]:bg-white-normal",
-        ],
+        "[&_.orbit-radio-icon-container]:has-[:checked]:border-2 [&_.orbit-radio-icon-container_span]:has-[:checked]:visible",
+        "[&_.orbit-radio-icon-container]:has-[:focus]:outline-blue-normal [&_.orbit-radio-icon-container]:has-[:focus]:outline [&_.orbit-radio-icon-container]:has-[:focus]:outline-2",
+        disabled
+          ? [
+              "cursor-not-allowed",
+              "[&_.orbit-radio-icon-container]:bg-cloud-light [&_.orbit-radio-icon-container]:border-cloud-dark",
```

Comment:
Is there any reason why these two lines are separated? Is it just "polish" thing 💅  or any technical reason behind?

Reply:
It's just a matter of readability and organization. While we don't have any standard within the team, I found it would be better to have classes that target other elements in separate lines